client/components/deposits-overview/utils.ts

```diff
@@ -29,7 +29,7 @@ export const getNextDeposit = (
 		return {
```

Comment:
Note that this entire file could be inlined into the only consumer of the `getNextDeposit` function: the `NextDepositDetails` component.

https://github.com/Automattic/woocommerce-payments/blob/d1384606930f7dcc2013804c0e51965e2cafc6f4/client/components/deposits-overview/next-deposit.tsx#L123

Most of this code will likely be removed when we redo the **Payments → Overview** UI as part of this project: issue TBC.

Reply:
Sounds like a great refactor. Side note, if we were to keep it, `pending` seems like a funny default, the most common would be `paid`.